include/ica_api.h

```diff
@@ -3005,6 +3002,71 @@ unsigned int ica_aes_xts(const unsigned char *in_data, unsigned char *out_data,
 			 unsigned int key_length, unsigned char *tweak,
 			 unsigned int direction);
 
+/**
+ * Encrypt or decrypt data with an AES key using the XEX Tweakable Bloc Cipher
```

Comment:
Block Cipher

Reply:
fixed. Was already wrong in the existing comment.....